src/msg/Policy.h

```diff
@@ -42,23 +42,26 @@ struct Policy {
   ThrottleType* throttler_messages;
   
   /// Specify features supported locally by the endpoint.
-  uint64_t features_supported;
+#ifdef MSG_POLICY_UNIT_TESTING
+  uint64_t features_supported{CEPH_FEATURES_SUPPORTED_DEFAULT};
+#else
+  static constexpr uint64_t features_supported{CEPH_FEATURES_SUPPORTED_DEFAULT};
+#endif
```

Comment:
Have you considered adjusting the unit tests instead of introducing the ifdef?  Looks like it's just one test case and we could handle "1. Suppose if only support less than required" case by requiring `RESERVED` (bit 62).

Reply:
Not necessarily as a part of this PR. I could agree the `ifdef` is far from being nice & elegant but I believe it sill is an improvement.

Reply:
I don't deny that this PR is an improvement but that future PR would be reverting a good chunk of this one.  If using `RESERVED` bit is feasible, it should be just a few of lines worth of adjustments and would save us the `ifdef`.